gradient/api_sdk/clients/experiment_client.py

```diff
@@ -367,7 +367,7 @@ def run_multi_node(
 
         :param str name: Name of new experiment  [required]
         :param str project_id: Project ID  [required]
-        :param str experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
+        :param int experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
```

Comment:
same as above, unless you parse input (which is string - GRPC or MPI), this is not int

Reply:
Fixing the comment